common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -93,7 +94,14 @@
     dirs = list(dirs)
     dirs.insert(0, theme.path / 'templates')
 engine = Engine(dirs=dirs)
-source, template_path = Loader(engine).load_template_source(path)
+loader = Loader(engine)
+
+for origin in loader.get_template_sources(path, dirs):
+    try:
+        source, template_path = loader.get_contents(origin), origin.name
+    except TemplateDoesNotExist:
+        pass
```

Comment:
The [previous implementation](https://github.com/django/django/blob/stable/1.8.x/django/template/loaders/filesystem.py#L33) raised `TemplateDoesNotExist` if the template wasn't found by the end of the loop, we should probably preserve that behavior.